setup.py

```diff
@@ -24,7 +24,8 @@
         "Jinja2",
         "markdown2",
         "pygments",
-        "simplepup"
+        "simplepup",
+        "pytest",
```

Comment:
This isn't needed in production, so use `tests_require=["pytest"],` for this, and then `python setup.py test`.

Reply:
Will do